### PR TITLE
Remove unneeded check for EnableWikiaHomePageCollections

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

**Comment by kvas-damian** [comment]
LGTM

File: extensions/wikia/WikiaHomePage/WikiaHomePageController.class.php [hunk 1]
```diff
@@ -233,33 +233,31 @@ protected function getList() {
...
-
-						if (!empty($collection['sponsor_hero_image'])) {
-							$processedCollection['sponsor_hero_image'] = $collection['sponsor_hero_image'];
-						}
+			$visualization = $this->getVisualization();
+			$collections = new WikiaCollectionsModel();
+			$collectionsList = $collections->getListForVisualization($this->wg->ContLang->getCode());
+
+			foreach ($collectionsList as $collection) {
+				if (count($collection['wikis']) == WikiaHomePageHelper::SLOTS_IN_TOTAL) {
```
Comment (Warkot) [inline extensions/wikia/WikiaHomePage/WikiaHomePageController.class.php]: Maybe swap `==` for `===`?

Comment (hakubo) [inline extensions/wikia/WikiaHomePage/WikiaHomePageController.class.php]: This code here works.
I do not want to introduce any logic changes. :)

Comment (Warkot) [inline extensions/wikia/WikiaHomePage/WikiaHomePageController.class.php]: That's what i thought. :+1:

**Comment by Warkot** [comment]
LGTM :shipit:

**Pull request merged**